src/lib/skilling/skills/mining.ts

```diff
@@ -250,6 +246,20 @@ const ores: Ore[] = [
 	}
 ];
 
+const MotherlodeMine = [
+	{
+		level: 30,
+		xp: 60,
+		id: 12_011,
+		name: 'Motherlode mine',
+		respawnTime: 5.5,
+		bankingTime: 60,
+		slope: 0.181,
+		intercept: 23.48,
+		petChance: 247_200
+	}
+];
+
```

Comment:
Why is this an array? I'm pretty sure I already commented on this in my last review

Reply:
Not sure on how you feel it would be best to have the details of this then, it made sense to me to follow a similar suit as other mine-ables.

Reply:
Remove the array, and make it just an object. MLM is only one thing, theres no purpose in making it an array. You dont have to try make it the same as ores.